### PR TITLE
Ask worker to not dry-run

### PR DESCRIPTION
**Pull request by JohanLorenzo**

### CODE REVIEW

**Comment by JohanLorenzo** [comment]
The production machines of pushapkworker are now up and running ([here for instance](https://tools.taskcluster.net/task-inspector/#bPP3XJ7YSRepfAt384qrBw/0)). As of now, the production machine doesn't publish anything yet (thanks to mozilla-releng/pushapkscript#2). With this PR, we'll allow the [prod instance of fennec-aurora-task-creator](https://dashboard.heroku.com/apps/fennec-aurora-task-creator) to ask the worker to commit the changes. As a consequence, the stage instance will keep dry running, whereas prod actually pushes live.

r? @rail

File: fennec_aurora_task_creator/config.py [hunk 1]
```diff
@@ -154,6 +156,10 @@ def _get_environment_or_config_or_default_value(config_from_json_file, path_list
 
     if value is None:
         raise MissingConfigurationError(environment_key, '/'.join(path_list))
+
+    if type(default_value) is bool and type(value) is not bool:
```
Comment (rail) [inline fennec_aurora_task_creator/config.py]: `isinstance()` is better to use here.

**Review by rail — APPROVED** [review]
A nit below.

File: fennec_aurora_task_creator/config.py (hunk 1 above)
Comment (JohanLorenzo) [reply fennec_aurora_task_creator/config.py]: Done. Thanks!

**Pull request merged**